src/pybind/CMakeLists.txt

```diff
@@ -1,62 +1,74 @@
 include(GNUInstallDirs)
 include(FetchContent)
+include(FindCUDAToolkit)
 
-if (USE_TORCH)
-  if (CMAKE_CUDA_COMPILER) # Include cuda buffers
-    set(module_source_files tensor_buffer_kernel.cu)
+# Configure CUDA
+if (USE_CUDA)
+  if(NOT DEFINED ${CMAKE_CUDA_ARCHITECTURES})
+    set(CMAKE_CUDA_ARCHITECTURES 50 52 60 61 70 75 80 86)
   endif()
-  # Import PyTorch's Python library to ensure correct linking
-  # Thanks to https://github.com/pytorch/pytorch/issues/38122
-  find_library(TORCH_PYTHON_LIBRARY torch_python PATHS "${TORCH_INSTALL_PREFIX}/lib")
-  message(STATUS "TORCH_PYTHON_LIBRARY: ${TORCH_PYTHON_LIBRARY} - ${PYTHON_INCLUDE_DIRS}")
-  message(STATUS "TORCH_LIBRARIES: ${TORCH_LIBRARIES} - ${TORCH_INCLUDE_DIRS}")
-  
+  find_package(CUDAToolkit REQUIRED)
+
+  enable_language(CUDA)
+  set(module_source_files "${module_source_files}" tensor_buffer_kernel.cu)
+  set(module_link_libraries CUDA::cudart)
+  set(module_compile_definitions USE_CUDA)
 endif()
 
+# Configure Inivation
 if (libcaer_FOUND) 
   set(module_compile_definitions WITH_CAER)
   set(module_source_files "${module_source_files}" usb.cpp)
```

Comment:
Is usb.cpp only used for inivation? If so, maybe it should be renamed to inivaton_usb.cpp or inivation.cpp

Reply:
Currently, yes. But it's intended to be for *any* USB input. At the moment we're automatically detecting the camera, so that can/should also happen for Prophesee (if the drivers are there).